strr-web/tests/unit/pages/rental-application.spec.ts

```diff
@@ -111,30 +115,34 @@ describe('Rental Application', () => {
     })
   })
 
+  // Utility function to filter only string values from an object
+  const filterValues = obj => Object.values(obj).filter(val => typeof val === 'string')
+
   it('should render Review and Confirm Step - Primary Contact Information section', async () => {
     wrapper = await mountSuspended(CreateApplication)
     await goToStep(5)
 
     const primaryContactReview = wrapper.findTestId('primary-contact-review')
     expect(primaryContactReview.exists()).toBe(true)
 
-    // check number of fields displayed in primary contact section
+    // Check number of fields displayed in primary contact section
     const primaryContactFields = primaryContactReview.findAll('[data-test-id=form-item]')
     expect(primaryContactFields).toHaveLength(9)
 
-    // update Property Manager state
+    // Update Property Manager state with mock data
     formState.primaryContact = mockPrimaryContact
     await nextTick()
 
-    // update Country from CA to Canada
+    // Update country from 'CA' to 'Canada' in mockPrimaryContact
```

Comment:
This comment is misleading, it does not update in mockPrimaryContact, rather returning a new updated object.